names.csv

```diff
@@ -1144,3 +1144,96 @@ zack,zach,zak
 zebedee,zeb
 zedediah,dyer,zed,diah
 zephaniah,zeph
+adam,adan 
+alberto,beto
+alejandro,alex,alexander
+alfredo,fredi,alfred
+andres,andrew
+anibal,hannibal
+antonio,tono,tony,anthony
+armando,nando,herman
+arturo, arthur
+bernardo,berna,bernard
+candelario,candel
+carlos,charles
+cristian,cris,christian
+cristobal,cris,christopher
+delfino,pillo
+domingo,mingo
+eduardo,lalo,edward
+enrique,quique,henry
+ernesto,neto,ernest
+esteban,steven
+federico,quiquo,kiko,federick
+fernando,nando,ferdinand
+francisco,paco,pancho,francis
+gelardo,gera,harold,gerald
+guadalupe,lupe,lupillo
+guillermo,memo,william
+hector,teto
+hipolito,polin,hippolytos
+humberto,beto,hubert
+ignacio,nacho,ignatitus
+jaime,james
+jesus,chuy,chucho
+jorge,koque,george

```

Comment:
I've noticed a few of these where the Spanish name has an equivalent English name. I'm on the fence if we want to include these or not. I think it makes sense if a common convention would be for a Jorge to go by George in a more English setting.